examples/nodejs/sites/update.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const { Site } = require('calibre')
+
+const updateSiteSettings = async () => {
+  const slug = 'calibre'
+  const name = 'Calibre Marketing Site'
+
+  // Update the site settings
```

Comment:
Is this what it does?

```suggestion
  // Rename the site
```